src/Swarm/TUI/Model/Achievement/Description.hs

```diff
@@ -0,0 +1,40 @@
+{-# LANGUAGE OverloadedStrings #-}
+
+module Swarm.TUI.Model.Achievement.Description where
+
+import Swarm.TUI.Model.Achievement.Definitions
+
+describe :: CategorizedAchievement -> AchievementInfo
```

Comment:
Would there be any benefit to having all this information stored in an external `.yaml` file?  I know we still need to have a data type enumerating the possible achievements, since the code for deciding when we have obtained them is arbitrary and can't live in an external data file.  But the titles, flavor text, explanation, etc. could all live in a data file.

Reply:
> we still need to have a data type enumerating the possible achievements

One of the members of this enumeration's sum type could have a string field, which makes it open-ended for achievements defined in config files.  Which raises the question, should all of the achievements be stored in config?  At the moment I think I prefer a hybrid: define as many achievements as feasible in code to obtain exhaustiveness-checking from the compiler, and let the rest be in `.yaml`.

Reply:
Makes sense, although if we allow achievements to be defined completely in a `.yaml` file, wouldn't we need some sort of little DSL to describe what the achievement is for?  Or are you imagining that achievements could be defined via some Swarm code just like goal conditions?

Reply:
Yes, they would be specially-annotated goals.  #795 is probably a prerequisite.